From 8053d79c56ede986c1d94b9a24d71c822a92be88 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20=C4=8Cerm=C3=A1k?= <dan.cermak@cgc-instruments.com>
Date: Sat, 4 Apr 2020 00:34:23 +0200
Subject: [PATCH] Drop the stdlib-shims subpackage

We neither need nor want the stdlib-shims package in Fedora.  It is a forward
compatibility package for older OCaml installations.  Patch it out instead.
Upstream does not want this patch until stdlib-shims is obsolete.
---
 alcotest.opam            | 1 -
 dune-project             | 1 -
 src/alcotest-engine/dune | 2 +-
 3 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/alcotest.opam b/alcotest.opam
index 756782f..3996bb0 100644
--- a/alcotest.opam
+++ b/alcotest.opam
@@ -25,9 +25,7 @@
   "astring"
   "cmdliner" {>= "1.2.0"}
   "re" {>= "1.7.2"}
-  "stdlib-shims"
   "uutf" {>= "1.0.1"}
-  "ocaml-syntax-shims"
   "odoc" {with-doc}
 ]
 conflicts: [
diff --git a/dune-project b/dune-project
index 9bccced..f634a31 100644
--- a/dune-project
+++ b/dune-project
@@ -29,9 +29,7 @@ tests to run.
   astring
   (cmdliner (>= 1.2.0))
   (re (>= 1.7.2))
-  stdlib-shims
-  (uutf (>= 1.0.1))
-  ocaml-syntax-shims)
+  (uutf (>= 1.0.1)))
  (conflicts
   (result (< 1.5)) ;; Ensure `Result.result` = `Stdlib.result` when possible
   (js_of_ocaml-compiler (< 5.8)) ;; requirement for the js stubs
diff --git a/src/alcotest-engine/dune b/src/alcotest-engine/dune
index 9c05d0a..65b33e7 100644
--- a/src/alcotest-engine/dune
+++ b/src/alcotest-engine/dune
@@ -25,6 +25,5 @@
   cmdliner
   fmt.cli
   re
-  stdlib-shims
   uutf)
  (preprocess future_syntax))
-- 
2.26.2
